assemblies/assembly_integrating-entando/main.adoc

```diff
@@ -1,34 +1,27 @@
-[id='assembly_guided-decision-tables']
+[id='assembly_integrating-entando']
 
 include::_artifacts/document-attributes.adoc[]
 
-= Integrating {PRODUCT} assets in Entando App Builder 5.0.3
+= Integrating {PRODUCT} assets in Entando App Builder
 // Context attribute is assembly specific and enables module reuse between assemblies.
 :doctype: book
 :imagesdir: _images
 :context: integrating-with-entando
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business processes developer, you can integrate the business assets you have created in {PRODUCT} with Entando App Builder 5.0.3. Entando provides an open-source, low-code Digital Experience Platform (DXP) for building modern web, mobile, and IoT applications. With Entando, you can create a custom user experience around your {PRODUCT} business assets and expand your application development strategy.
+As a business processes developer, you can integrate the business assets you have created in {PRODUCT} with Entando App Builder. The App Builder provides an open-source, low-code Digital Experience Platform (DXP) for building modern web, mobile, and IoT applications. With the App Builder, you can create a custom user experience around your {PRODUCT} business assets and expand your application development strategy.
```

Comment:
"The App Builder"  >  "App Builder" No "the". So "App Builder provides ...", "With App Builder...", etc. Search all instances and update.

Reply:
Done